src/emulator/controller.ts

```diff
@@ -122,23 +127,26 @@ export async function startAll(options: any): Promise<void> {
     await startEmulator(functionsEmulator);
   }
 
-  if (targets.indexOf(Emulators.FIRESTORE) > -1) {
+  if (shouldStart(options, Emulators.FIRESTORE)) {
     const firestoreAddr = Constants.getAddress(Emulators.FIRESTORE, options);
 
     const args: FirestoreEmulatorArgs = {
       host: firestoreAddr.host,
       port: firestoreAddr.port,
     };
 
-    const rules: string = path.join(options.projectRoot, options.config.get("firestore.rules"));
-    if (fs.existsSync(rules)) {
-      args.rules = rules;
-    } else {
-      utils.logWarning(
-        `Firestore rules file ${clc.bold(
-          rules
-        )} specified in firebase.json does not exist, starting Firestore emulator without rules.`
-      );
+    const rulesLocalPath = options.config.get("firestore.rules");
+    if (rulesLocalPath) {
```

Comment:
Any chance we want to log something here to say that because the rules file isn't set that the local emulator will function diff than the db in prod?